health_check/contrib/redis/backends.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from django.conf import settings
+from kombu import Connection, exceptions
```

Comment:
`kombu` is still not listed as a requirement. I also don't think that you need `kombu` since it's a messaging library. I would prefer to use plain [redis](https://pypi.org/project/redis/).

Reply:
Redis library implemented instead of kombu